query/common/src/main/java/org/codice/ddf/admin/common/fields/base/BaseObjectField.java

```diff
@@ -69,29 +65,13 @@ public void setValue(Map<String, Object> values) {
         }
 
         validationErrors.addAll(getFields().stream()
-                .filter(field -> field instanceof DataType)
-                .map(field -> (List<ErrorMessage>) ((DataType)field).validate())
+                .map(field -> (List<ErrorMessage>) field.validate())
```

Comment:
`Incompatible types. Required Collection<? extends ErrorMessage> but 'collect' was inferred to R: no instance(s) of type variable(s) exist so that Object conforms to ErrorMessage inference variable T has incompatible bounds: lower bounds: Object upper bounds: Object, ErrorMessage`